CorsixTH/Lua/map.lua

```diff
@@ -169,30 +169,22 @@ the original game levels are considered.
 has been loaded.
 ]]
 function Map:load(level, difficulty, level_name, map_file, level_intro, map_editor)
-  local objects, _
+  local objects, _, result
```

Comment:
Please avoid introducing new variables miles before you use them.

Introduce it just above line 207 (that seems the first use of it).